### DISCUSSION TITLE
Installation without Operator

### DISCUSSION THREAD

**Discussion by darktempla** [strimzi/strimzi-kafka-operator#7133] [General]
Hi all, so I have hopefully not such a unique position.

At home I run a arm64 only Kubernetes cluster and as such I have found getting the OLM operator up and running with arm64 support not possible at this stage due to the catalog not playing nicely - https://github.com/operator-framework/operator-lifecycle-manager/issues/2823.

I first did a check and was happy to see you support arm64 as per - https://github.com/strimzi/strimzi-kafka-operator/issues/3357

So that posses my question to the community, is there a way to install Strimzi and it's CRDs and get things up and running without using operators. I appreciate the benefits of them and use them at work with OpenShift just not possible at this stage with my home setup.

Apologies in advance if I have overlooked something in the doco.

**Comment by scholzj** [comment]
Strimzi is based around the operator. So you always need to install the operator. That said, OLM is just one of the many ways to install the operator. You can also just download the ZIP files with the YAML files and install them with `kubectl` or you can use Helm etc. More info is in the docs: https://strimzi.io/docs/operators/latest/full/deploying.html#con-strimzi-installation-methods_str

So you need to install the operator, but if OLM does not work on your home setup, just pick another method.

PS: I tested the OLM submission for Strimzi 0.30.0 on my ARM based Minikube on MacOS and it seemed to work fine for me there.

**Reply by darktempla** [reply]
@scholzj - thanks heaps mate for the speedy reply. That's great, even better you have had some success on ARM always a bit of roll the dice exercise atm however the industry is getting better at supporting ARM.
